bin/hbase

```diff
@@ -493,7 +493,9 @@ add_jdk11_deps_to_classpath() {
 }
 
 add_opentelemetry_agent() {
-  if ! agent_jar=$(find lib/trace -type f -name "opentelemetry-javaagent-*" 2>/dev/null); then
+  if [ -e "${OPENTELEMETRY_JAVAAGENT_PATH}" ] ; then
+    agent_jar="${OPENTELEMETRY_JAVAAGENT_PATH}"
+  elif ! agent_jar=$(find "${HBASE_HOME}/lib/trace" -type f -name "opentelemetry-javaagent-*" 2>/dev/null); then
```

Comment:
Do you want `find -L`? I'm not sure if we have already resolved `HBASE_HOME` to be the "real" path by the time this function is called.

Reply:
Umm, maybe? It's true that we care about the thing the link resolves to. let me try it with `-L`, see if there's a difference. FYI, the above code is working fine for me...

Reply:
Yeah, I would expect for normal environments, your existing command would work just fine. I do know that Cloudera's current offering would break if we haven't fully resolved $HBASE_HOME (using client-facing-thirdparty as an analogy)
```
[root@cod--vy18bale6smv-worker2 ~]# find  /opt/cloudera/parcels/CDH/lib/hbase/lib/client-facing-thirdparty -type f -name 'slf4j*'
[root@cod--vy18bale6smv-worker2 ~]# find -L /opt/cloudera/parcels/CDH/lib/hbase/lib/client-facing-thirdparty -type f -name 'slf4j*'
/opt/cloudera/parcels/CDH/lib/hbase/lib/client-facing-thirdparty/slf4j-log4j12-1.7.30.jar
/opt/cloudera/parcels/CDH/lib/hbase/lib/client-facing-thirdparty/slf4j-api-1.7.30.jar
```

It looks like hbase-config.sh only sets HBASE_HOME based on the value from `dirname $0` (where $0 is bin/hbase). Probably not a big deal either way, but you might save me a headache later with a `find -L` (and I believe it to be zero-risk).

Reply:
Things seem to work fine on my end with your `-L` addition.